dbt/adapters/snowflake/connections.py

```diff
@@ -286,6 +291,7 @@ def connect():
                 client_session_keep_alive=creds.client_session_keep_alive,
                 application="dbt",
                 insecure_mode=creds.insecure_mode,
+                session_parameters=session_parameters,
                 **creds.auth_args(),
             )
 
```

Comment:
Shouldn't we also remove the below code? 
```
if creds.query_tag:
                handle.cursor().execute(
                    ("alter session set query_tag = '{}'").format(creds.query_tag)
                )
```

Reply:
removed this seemingly duplicated logic and tested locally, didn't seem to cause any issues or leave leftover tests in snowflake by removing